### DISCUSSION TITLE
Drag and Drop Between Widgets and/or Containers

### DISCUSSION THREAD

**Discussion by BillBuilt** [fyne-io/fyne#4757] [Q&A]
I have read some discussions about drag & drop (DnD) but they seem to be mainly concerned with the notion of dropping files onto the app. I'd like to use DnD from within the app. Is it currently possible to drag an item from a widget and drop it onto another widget or container? For example, could a list item be dragged and dropped onto another container?

**Comment by andydotxyz** [comment]
By building an overlay into your app where items can be shown during a drag you can do just that.
I built a demo for a chess app where the drag moves pieces. https://youtu.be/zlPDWBLhn6c?si=tAXU3PaQcKcA6wcQ
